src/sync/pull.rs

```diff
@@ -255,6 +257,7 @@ pub struct PullRequest {
     pub cookie: String,
     #[serde(rename = "lastMutationID")]
     pub last_mutation_id: u64,
+    #[serde(rename = "pullVersion")]
```

Comment:
For consistency but we should rename all of them to snake_case. I added an entry to the task list.

https://www.w3.org/TR/design-principles/#casing-rules

Reply:
Is this really followed? Seems icky at least for js consumers because then the resulting js objects conflict with js style.

I personally prefer it as is but if you can show me that everyone else is using snake case I’ll go along.

Reply:
The JS API is still all camelCase. This is just for the json.

https://developer.mozilla.org/en-US/docs/Web/Manifest
https://developer.chrome.com/docs/extensions/mv3/manifest/

Reply:
Strong preference for narrow gully hungarian notation, which would make this version integer `VIpULLvERSION`